python/sanlock.c

```diff
@@ -1738,6 +1738,36 @@ __module_init(PyObject* m)
     return 0;
 }
 
+#if PY_MAJOR_VERSION >= 3 /* Python 3 module init */
+
+static struct PyModuleDef moduledef = {
+        PyModuleDef_HEAD_INIT,
+        MODULE_NAME,
+        pydoc_sanlock,
+        -1,
+        sanlock_methods,
+};
+
+PyMODINIT_FUNC
+PyInit_sanlock(void)
+{
+    PyObject *py_module;
+
+    py_module = PyModule_Create(&moduledef);
+
+    if (py_module == NULL)
+        return NULL;
+
+    if (__module_init(py_module)) {
+        Py_DECREF(py_module);
+        return NULL;
+    }
+    /* Success */
```

Comment:
This comment is obvious, can be removed.

Reply:
solved in PR #3